ChemistryLib/PhreeqcKernel.cpp

```diff
@@ -48,8 +50,24 @@ PhreeqcKernel::PhreeqcKernel(std::size_t const num_chemical_systems,
     }
     use.Set_solution_in(true);
 
+    // For equilibrium-controlled reactions
+    if (pure_phase_assem)
```

Comment:
Maybe `equilibrium_reactions` would be good name.
Anyway don't shorten the assemblage.

Reply:
Renamed.